src/modules/core/components/emoji/emoji.tsx

```diff
@@ -0,0 +1,22 @@
+import { type FC } from 'react';
+// animations
+import Lottie from 'lottie-react';
+import sunglasses from '../../../../assets/animations/emoji_sunglasses.json';
+import heartEyes from '../../../../assets/animations/emoji_heart_eyes.json';
+
+import type { EmojiProps } from './emoji.interface';
+
+// keys
+const emojiAnimations: Record<'sunglasses' | 'heart_eyes', unknown> = {
```

Comment:
Use `EmojiName` that is described above - `Record<EmojiName, unknown>`. And probably we can change `unknown` to `string`.